toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -86,31 +79,52 @@ func updateFstabForVerity(verityList []imagecustomizerapi.Verity, imageChroot *s
 	return nil
 }
 
-func prepareGrubConfigForVerity(imageChroot *safechroot.Chroot) error {
-	bootCustomizer, err := NewBootCustomizer(imageChroot)
-	if err != nil {
-		return err
-	}
+func prepareGrubConfigForVerity(verityList []imagecustomizerapi.Verity, imageChroot *safechroot.Chroot) error {
+	for _, verity := range verityList {
+		mountPath := verity.FileSystem.MountPoint.Path
 
-	err = bootCustomizer.PrepareForVerity()
-	if err != nil {
-		return err
-	}
+		bootCustomizer, err := NewBootCustomizer(imageChroot)
```

Comment:
You can probably pull the `NewBootCustomizer` and `bootCustomizer.WriteToFile` calls out of the for loop.